TFT/src/User/SanityCheck.h

```diff
@@ -51,6 +51,18 @@ extern "C" {
   #define TERMINAL_COLOR_SCHEME 0
 #endif
 
+#ifdef PROGRESS_BAR_COLOR
+  #if PROGRESS_BAR_COLOR > 9
+    #error "PROGRESS_BAR_COLOR cannot be greater than 9"
+  #endif
+
+  #if PROGRESS_BAR_COLOR < 0
+    #error "PROGRESS_BAR_COLOR cannot be less than 0"
+  #endif
+#else
+  #define PROGRESS_BAR_COLOR 0
+#endif
+
```

Comment:
same as in Configuration.h. It should be moved after TERMINAL_KEYBOARD_LAYOUT

Reply:
Done.